README.md

```diff
@@ -170,44 +170,159 @@ convert().list('mass');
 Custom Measures
 ---------------
 
+It's possible to define custom measures if the ones packaged aren't what you need:
```

Comment:
I think this generic example is fine (in lieu of a better concrete example) but it might be worth to add a bit more description and possibly additional inline comments to help people who are new to the code. The description could be extended to explain exactly what this example does and what definitions are required to make it happen. I know when I first tried to add my own measure I was quite confused about exactly what I needed to write. I think just a couple more sentences description could go a long way.

Reply:
Thanks for the feedback! I'll make the changes you've suggested in the next couple days.